contracts/profile/src/credits.rs

```
@@ -49,11 +72,20 @@ pub fn spend(
         idempotency::mark_seen(env, &op_id);
         return Ok(());
     }
+    
 
     let mut profile = storage::get_profile(env, &user).ok_or(Error::ProfileNotFound)?;
```

Clippy (1.96.0)

[warning] variable does not need to be mutable

(warning)